opentracing/src/main/java/opentracing/Span.java

```diff
@@ -25,16 +25,7 @@
   /**
    * Suitable for serializing over the wire, etc.
    */
-  TraceContext traceContext();
-
-  /**
-   * Denotes the beginning of a subordinate unit of work.
-   *
-   * @param operationName name of the operation represened by the new span from the perspective of
-   * the current service.
-   * @return a new child Span in "started" state.
-   */
-  Span startChild(String operationName);
+  SpanContext childContext();
```

Comment:
> I suppose I inferred that SpanContext childContext() was still an operation to create a child span?

No. The use of `childContext()` is better explained in https://github.com/opentracing/opentracing-java/pull/7#discussion_r50500237

The creation of child Spans is still done via 
 `Tracer.joinTrace("new-span-name", currentSpan.childContext());`